charts/barman/templates/configmap-barman-backups.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
     {{- include "barman.labels" . | nindent 4 }}
 data:
   {{- range .Values.barman.backups }}
-  {{ .postgresql.host }}.conf: |
-      [{{ .postgresql.host }}]
+  {{ .scopeName }}.conf: |
+      [{{ .scopeName }}]
```

Comment:
What is the benefit of using `scopeName` instead of the hostname?

Reply:
Hi @tongpu 
When I used `.postgresql.host` in my configmaps, I got an error that Barman can't find `pg` which is the `server_name` in Barman commands. Also, Kubernetes domain names for postgresql in my cluster are too long, so it's easier to work with `pg`. Therefore I replaced `.postgresql.host` by `scopeName` in `configmap-barman-backups` and `configmap-barman-cron`.

Reply:
I think the current state of things was introduced with #631 and it broke quite a few use-cases.

Reply:
Hi @hairmare 
I checked this PR. As I said, when I used `.postgresql.host`, it caused an error implying that Barman can't find server `pg`. I'm not aware that how other friends use `.postgresql.host` and perform backup since Barman is looking for `pg` and I didn't find anything to change this. So I let Barman use `pg` and put it in my configmaps. Sounds right?